src/yze-combat.js

```diff
@@ -64,8 +64,30 @@ Hooks.once('ready', async () => {
   // TODO Hooks.call('yzeCombatReady');
 
   console.log('YZEC | READY!');
+
+  // TODO Remove this example before merge
+
+  // This listens for a message from the client called when the user clicks the slow button in the combat tracker.
+  Hooks.on(`${MODULE_NAME}.slow-action-button-clicked`, data => {
+    console.log('YZEC | Event', data);
+    // The hook supplies a socket emit function that can be used to send a message to the server/other clients.
+    data.emit({ forGmOnly: 'secret lover' });
+  });
+  // This listens for a message from the client called when the user clicks the Duplicate button in the combat tracker.
+  Hooks.on(`${MODULE_NAME}.combat-tracker-duplicate-clicked`, data => {
```

Comment:
```suggestion
  Hooks.on(`${MODULE_NAME}.combat-tracker-duplicate-button-clicked`, data => {
```
Suggestion for staying consistent with the above one.